### PR TITLE
log improvement for ocm.py

### PR DESCRIPTION
**Pull request by mwaykole**
Signed-off-by: mwaykole <mwaykole@redhat.com>

### CODE REVIEW

File: utils/scripts/util.py [hunk 1]
```diff
@@ -59,20 +59,23 @@ def read_yaml(filename):
...
-            check=True,
             stdout=subprocess.PIPE,
             stderr=subprocess.PIPE,
-            universal_newlines=True,
+            text=True
         )
+
         output = process.stdout
+        if stderrPrint and process.stderr:
+            print("ods-ci-Error:{}".format(stderrPrint.stderr, end=""))
```
Comment (bdattoma) [inline utils/scripts/util.py:77]: shouldn't it be `process.stderr` instead of `stderrPrint.stderr`?

Comment (mwaykole) [reply utils/scripts/util.py:77]: correct  ill replace it thanks :)

**Review by bdattoma — DISMISSED** [review]

**Review by aloganat — DISMISSED** [review]
lgtm

**Comment by mwaykole** [comment]
[#1017](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-smoke/2017/console)  RUNNING

**Comment by mwaykole** [comment]
negative testing also works fine 

`└─(10:48:09 on logimproment ✭)──> python3 utils/scripts/ocm/ocm.py create_cluster --aws-account-id tat --aws-accesskey-id tat --aws-secret-accesskey tat --cluster-name qeaisrhods-umne --aws-region us-east-2 --aws-instance-type m5.2xlarge --num-compute-nodes 2 --openshift-version 4.11-latest

ods-ci-logger: INFO     CMD: ocm list versions  | grep -w "4\.11*"
ods-ci-logger: INFO     CMD: ocm create cluster --aws-account-id tat --aws-access-key-id tat --aws-secret-access-key tat --ccs --region us-east-2 --compute-nodes 2 --compute-machine-type m5.2xlarge --version 4.11.26  qeaisrhods-umne
ods-ci-Error: Error: status is 400, identifier is '400', code is 'CLUSTERS-MGMT-400' and operation identifier is '7845e768-d3df-431e-8e0a-4a511878066f': Failed to get list of available aws cloud regions.

ods-ci-logger: INFO     Failed to create osd cluster qeaisrhods-umne
`

**Review by aloganat — APPROVED** [review]
lgtm

**Review by tarukumar — APPROVED** [review]
lgtm

**Pull request merged**